testkit/dbtestkit.go

```diff
@@ -60,6 +83,14 @@ func (tk *DBTestKit) MustQuery(sql string, args ...interface{}) *sql.Rows {
 	return rows
 }
 
+// MustQueryRows query the statements
+func (tk *DBTestKit) MustQueryRows(query string, args ...interface{}) {
+	rows := tk.MustQuery(query, args...)
+	tk.require.True(rows.Next())
+	tk.require.NoError(rows.Err())
+	rows.Close()
```

Comment:
You should check the error return here I think. @yedamao could you submit a follow up to fix it?